dashboard/test/models/concerns/text_to_speech_test.rb

```diff
@@ -54,7 +64,9 @@ class TextToSpeechTest < ActiveSupport::TestCase
     assert_equal @level_with_markdown_instructions_override.tts_path(@level_with_markdown_instructions_override.tts_markdown_instructions_text), 'sharon22k/180/100/25352162fcc2d1e5c4ea3f91b9b39c3f/.mp3'
   end
 
-  test 'sanitize html' do
-    assert_equal @level_with_raw_html.tts_markdown_instructions_text, "This should have  no excess formatting \n"
+  test 'sanitize html and xml' do
+    assert_equal @level_with_raw_html.tts_markdown_instructions_text, "This should have  no excess formatting\n"
+    assert_equal @level_with_block_html.tts_markdown_instructions_text, "This block should get stripped:\n"
+    assert_equal @level_with_xml.tts_markdown_instructions_text, "This block should get stripped:\n"
```

Comment:
Depends on how familiar you are with Redcarpet 😄 

Two newlines in the input are necessary to separate paragraphs and to treat the xml as its own "block", as is standard for markdown input.

Newlines within the content itself are irrelevant, as is standard for markdown output, but for some reason one is always added to the end.